finetuner/finetuner.py

```diff
@@ -27,6 +27,16 @@ def login(self):
         hubble.login()
         self._init_state()
 
+    def notebook_login(self, force: bool = False):
+        """Log in to Hubble account, initialize a client object
+        and create a default experiment.
+
+        :param force: If set to true, overwrite token and re-login.
```

Comment:
Can you add the force parameter also to the login function. It also exists there.